trade-finance-manager-api/api-tests/v1/middleware/api-rate-limiting.api-test.js

```diff
@@ -0,0 +1,33 @@
+require('dotenv').config();
+const { generateApp } = require('../../../src/generateApp');
+const createApi = require('../../api');
+
+describe('api rate limiting', () => {
+  const rateLimit = process.env.RATE_LIMIT_THRESHOLD;
+
+  let app;
+  let as;
+  let sendRequestTimes;
+
+  beforeEach(() => {
+    app = generateApp();
+    ({ as } = createApi(app));
+    sendRequestTimes = (numberOfRequestsToSend) => Promise.allSettled(Array.from({ length: numberOfRequestsToSend }, () => as(null).put(null).to('/v1/deals/submit')));
+  });
+
+  it('returns a 429 response if more than RATE_LIMIT_THRESHOLD requests are made from the same IP to the same endpoint in 1 minute', async () => {
```

Comment:
Over 60 seconds assertion?

Reply:
We've discussed with @abhi-markan and decided to delay adding this test until after resolving the other SR issues this week